libs/remix-ui/workspace/src/lib/remix-ui-workspace.tsx

```diff
@@ -937,6 +937,14 @@ export function Workspace() {
                     <label className="pl-2 form-check-label" style={{wordBreak: 'keep-all'}}>
                       <FormattedMessage id='filePanel.workspace' />
                     </label>
+                    {selectedWorkspace && selectedWorkspace.name === 'code-sample' && <CustomTooltip
+                      placement="right"
+                      tooltipId="saveCodeSample"
+                      tooltipClasses="text-nowrap"
+                      tooltipText={<FormattedMessage id="filePanel.saveCodeSample" />}
+                    >
+                      <i onClick={() => renameCurrentWorkspace()} className="far fa-exclamation-triangle text-info ml-2 mt-1" aria-hidden="true"></i>
```

Comment:
I think the exclamation should be either be text-warning or danger. In dark mode, it's contrast isn't great. The size could be a bit bigger though.

Reply:
please use align-self-center instead of mt-1.
I agree with Joe, it should rather be text-warning.